react/react2/week1/github-user-search/src/GitSearchComps/ListItems.js

```diff
@@ -0,0 +1,28 @@
+/*
+This function has access to users state in global state and 
+conditionally return a list of users or loading-no result and error message. 
+*/
+
+import React, { useContext } from 'react'
+import { GlobalContext } from './GlobalState'
+
+function ListItems() {
+    
+    const GlobalItems = useContext(GlobalContext);
+    const users = GlobalItems.users.item;
+    const loading = GlobalItems.users.loading;
+    const error = GlobalItems.users.error;
+    const emptyResult=GlobalItems.users.emptyResult;
+
+
+  return (
+      <div>
+          {loading && <p>loading...</p>}
+          {emptyResult  && <p>No Result</p>}
+          {error !== '' && <p>{error}</p>}
+          {users.map((user) =><h4> {user.login}</h4>)}
```

Comment:
I noticed you are getting a "key" warning, that's because you are not passing a key prop to the h4 element. You can read more on that here https://reactjs.org/docs/lists-and-keys.html

Reply:
Hello , I have fixed the key warning using the `user.id`, about the styling , I didn't spend time on it , because I have to work on the individual project(meal-sharing ). If it is ok , I will merge the PR ? :)